### ISSUE TITLE
Tutorials are required

### ISSUE THREAD

**Issue by SilverFoxA** [ksoichiro/Android-ObservableScrollView#116]
Hello sir,
The tutorials or guide that's provided with the library files is not enough. That doesn't even tells completely on how to use the following feature like "Viewpager&tab&scrollview"

**Comment by ksoichiro** [comment]
Hi, thank you for your feedback.
I'm going to put some tutorials to `docs` directory.
Also, I'm preparing a website for it.
http://ksoichiro.github.io/Android-ObservableScrollView/
Please wait until it's completed.

**Comment by SilverFoxA** [comment]
Thank you a lot

**Comment by petersnoopy** [comment]
Great job!!!

**Comment by JustLey** [comment]
I agree some tutorials could help, but I didn't have a problem implementing and/or modifying your library. By the way, congratulations, your library is extremely well made in my opinion. You even cared about the coreography of the elements.
